.github/workflows/add-labels.yml

```diff
@@ -0,0 +1,17 @@
+name: add-labels
+on: 
+  workflow_dispatch:
```

Comment:
should this be a cron schedule?

Reply:
No it's a one-time thing, this is just to create new labels with particular colours.

Reply:
Oh! I thought it will automatically add a label as well :D